crates/bindings/tests/ui/tables.stderr

```diff
@@ -129,11 +129,11 @@ error[E0277]: `&'a Alpha` cannot appear as an argument to an index filtering ope
              &ConnectionId
              &Identity
              &Lifecycle
-             &RawMiscModuleExportV9
```

Comment:
@Centril UI tests were failing until I updated this file. I believe it's because the trait FilterableValues was auto-implemented for RawMiscModuleExportV9, but this PR changes the enum to include a variant with data, which affects the generated messages.